cli/src/main/java/com/devonfw/tools/ide/version/VersionIdentifier.java

```diff
@@ -205,7 +205,9 @@ public String toString() {
    */
   public static VersionIdentifier of(String version) {
 
-    if (version.equals("latest")) {
+    if (version == null) {
```

Comment:
What about empty versions? Possible at all?

Reply:
empty String results in `null` because `VersionSegment.of("")` return `null`.
Can we close this review comment?
If you have questions it is better to ask via chat than gitub review comment.